pkg/apimachinery/announced/group_factory.go

```diff
@@ -156,6 +156,15 @@ func (gmf *GroupMetaFactory) Register(m *registered.APIRegistrationManager) erro
 	prioritizedVersions = append(prioritizedVersions, unprioritizedVersions...)
 	m.RegisterVersions(prioritizedVersions)
 	gmf.prioritizedVersionList = prioritizedVersions
+
+	// Register empty group. Will be filled with proper values in Enable()
+	groupMeta := &apimachinery.GroupMeta{
```

Comment:
Also note the current semantics: it's called `Register()`, but does not register the group (only the versions). Group registration is done in `Enable()`. That's what I want to change.

@lavalamp ^^